### DISCUSSION TITLE
Raw resource conditionally on platform

### DISCUSSION THREAD

**Discussion by daltzctr** [dotnet/maui#16891] [Q&A]
I have the following resource:

`test.zip` that I want to be bundled with the application on Windows, but **not** Android or iOS. What's the best way to do that? I'd rather not resort to an embedded asset, as that has other issues.

**Comment by xtuzy** [comment]
try
```
<ItemGroup Condition="$(TargetFramework.Contains('-windows')) == true">
		<MauiAsset Include="Resources\test.zip" LogicalName="%(RecursiveDir)%(Filename)%(Extension)" />
	</ItemGroup>
```
